Gemfile

```diff
@@ -8,8 +8,10 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
 gem 'bootsnap', '>= 1.4.2', require: false
 gem 'rack-cors', require: 'rack/cors'
+gem 'rest-client'
```

Comment:
Since gems are being added be sure to include them in the PR description. The format is pinned in the group channel I think.

Reply:
added the gems to the PR